release/src/router/www/state.js

```diff
@@ -992,7 +992,7 @@ function show_banner(L3){// L3 = The third Level of Menu
  	}
 
 	// dsl does not support operation mode
-	if (!dsl_support && !lyra_hide_support) {
+	if ((based_modelid == "DSL-AC68U") || (!dsl_support && !lyra_hide_support)) {
```

Comment:
would removing dsl_support work?
We can enable this for DSL-AX82U as far as I konw there are no other DSL models.

Reply:
yes, it will, if you prefer I can make edit it again removing that flag, but I don't know if on other modem it can cause problems and I can't test

Reply:
I will fix this. The DSL models are the following ones and I think they are fine.
I wanna test this on my DSL-AX82U too! 
router/www/Advanced_WAN_Content.asp:	if(dsl_support)      //MODELDEP: DSL-AC68U,DSL-AC68R,DSL-AX82U